Milestone 5# Add stylings

gscarv13 opened the pull request:
### This pull request contains the following:

- Added Stylings to the BookForm component
- Added Stylings to the book component
- Added and Styled a Placeholder Navigation bar

gscarv13 committed:
Add css rules to BookForm

gscarv13 committed:
Update Book and BookList JSX structure, Add initial Book css

gscarv13 committed:
Fix BookForm margin top, Finish Book css

gscarv13 committed:
Add Navbar

gscarv13 committed:
Make progress placeholder random, Modify filter css

gscarv13 committed:
Fix Stylelint error

afanimartin reviewed (changes requested):
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this PR. You have managed to fulfill most of the requirements for this milestone. Well done!
## Changes 🔕 🔕 🔕 
- update the readme. add the exact command for cloning the repo in the `setup and install` section
- add the property `border-radius: 50%` to the user avatar. this will improve its design and will make it look like the given design
- the filter input above the list of books should be removed. the original design doesn't have it
- the dimensions for the input fields for creating a new book in the original design are: `664x45, 285x45, 184x45` while your dimensions are: `408x45, 285x45, 184x45`. make sure you follow the exact dimensions as the original


### Martin Afani
#### Slack: whiz25

gscarv13 committed:
Update README instructions

gscarv13 committed:
Fix user button

gscarv13 committed:
Fix BookForm component input dimensions

gscarv13 committed:
Move filter select to the Navbar

gscarv13 commented:
### Thank You for your review @afanimartin!

As requested, I fixed all UI bugs, along with the README and the FilterComponent.

Due to a lack of specificity, I moved the filter to the Navigation bar :v:

rukundoeric reviewed (approved):
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight 
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯
  - App is functional and available online. 💯 
  - Issues raised from the previous review are all addressed. 💯 
  

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/),
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**
